app/presenters/track.rb

```diff
@@ -47,17 +47,17 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        File.
-          read("./x/docs/md/track/BETTER.md").
-          gsub('REPO', trackler_track.repository).
-          gsub('TOPIC', topic_key.to_s.upcase).
-          gsub('EXT', trackler_track.doc_format)
+        content_with_trackler_data("./x/docs/md/track/BETTER.md").gsub('TOPIC', topic_key.to_s.upcase)
       end
 
       def fallback_topic_content(topic_key)
         filepath = "./x/docs/md/track/#{topic_key.upcase}.md"
         return '' unless File.exist?(filepath)
-        File.read(filepath)
+        content_with_trackler_data(filepath)
+      end
+
+      def content_with_trackler_data(filepath)
+        File.read(filepath).gsub('REPO', trackler_track.repository).gsub('EXT', trackler_track.doc_format)
```

Comment:
Another round of, this is good, but what about:

The `File.read` has now been decoupled from the `File.exist?` I'd like to see these much closer too each other.

Then the template replacer isn't reading the file AND doing the template replacements, so can take the already loaded data as an argument: `track_specific_template_replacements(file_data)`

I've chosen this name because I want to have an idea of a) something is being replaced, b) what that might be.

Another way to do this would be passing in the trackler track so we get a clue about what it might be `template_replacements(file_data, trackler_track)`. 

But it doesn't matter a whole lot in this case which solution you prefer.